dbt/parser.py

```diff
@@ -448,6 +448,37 @@ def load_and_parse_macros(package_name, root_project, all_projects, root_dir,
     return result
 
 
+def get_parsed_schema_test(test_node, test_type, model_name, config,
+                           root_project, projects, macros):
+
+    package_name = test_node.get('package_name')
+    test_namespace = None
+    split = test_type.split('.')
+
+    if len(split) > 1:
+        test_type = split[1]
+        package_name = split[0]
+        test_namespace = package_name
+
+    source_package = projects.get(package_name)
+    if source_package is None:
+        dbt.exceptions.raise_compiler_error(
+                'Error while parsing "{}" test on model "{}".\nThe required '
+                'package "{}" was not found. Is the package installed?'.format(
```

Comment:
mention running dbt deps (Is the package installed? Hint: You may need to run `dbt deps`.)

Reply:
@cmcarthur I thought about doing this, but figured we were going to change `dbt deps` to `dbt package --install` (or similar) in the future and felt weird about writing out `dbt deps` here.

Good idea though, changing

Reply:
Updated:

```
$ dbt test
Encountered an error:
Compilation Error in test schema (models/schema.yml)
  Error while parsing "dbt_utils.unique" test on model "test".
  The required package "dbt_utils" was not found. Is the package installed?
  Hint: You may need to run `dbt deps`.
```